### ISSUE TITLE
symfony/serializer 5.3

### ISSUE THREAD

**Issue by r3sist** [canihavesomecoffee/theTVDbAPI#18]
Hello canihavesomecoffee,
is it possible to make compatible this package with Symfony serializer component v5? I have not found any mentions about deprecations or BC issues in Symfony blog or documentation. And I would upgrade my app to Symfony framework 5.3.

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires canihavesomecoffee/thetvdbapi 2.* -> satisfiable by canihavesomecoffee/thetvdbapi[2.0.0.beta].
    - canihavesomecoffee/thetvdbapi 2.0.0.beta requires symfony/serializer ^4.0 -> found symfony/serializer[v4.0.0-BETA1, ..., 4.4.x-dev] but these were not loaded, likely because it conflicts with another require.
```

**Comment by canihavesomecoffee** [comment]
I can take a look at it when I'm back from holidays (in about two weeks).

If there's no deprecated calls or BC issues, you could try it out yourself with a fork (and definitely be welcome to open a PR for it!) and see what it gives.

**Comment by canihavesomecoffee** [comment]
@r3sist, I brought out a new beta with minimal support for the new API. Could you test this this week if you have time and provide some feedback if you encounter issues? The version of symfony components were bumped to 5.3.

**Comment by r3sist** [comment]
@canihavesomecoffee Sorry for the late answer. I couldn't test it last week, but I will try. Thank you for your work.

**Comment by canihavesomecoffee** [comment]
I should've picked it up sooner too, especially when they announced shutdown of the V3 API... So I'll (have to) release a V4 compatible one with minimal support rather than full V4 compatibility.

**Comment by canihavesomecoffee** [comment]
Closed by release of v2.0.0.0
